Library/Formula/euslisp.rb

```diff
@@ -0,0 +1,39 @@
+class Euslisp < Formula
+  desc "EusLisp is an integrated programming system for the research on intelligent robots"
```

Comment:
You can shorten this: `desc "integrated programming system for the research on intelligent robots"`.